privacyidea/lib/eventhandler/tokenhandler.py

```diff
@@ -242,12 +242,12 @@ def do(self, action, options=None):
                 log.info("{0!s} for token {1!s}".format(action, serial))
                 if action.lower() == ACTION_TYPE.SET_TOKENREALM:
                     realm = handler_options.get("realm")
-                    only_realm = handler_options.get("only_realm")
+                    only_realm = bool(handler_options.get("only_realm"))
```

Comment:
Minor issue: If the handler option is set to ``0``, ``only_realm`` still is set to ``True`` (because ``bool("0") == True``). This can happen if the user sets the option (and saves) and then subsequently unsets it.
I guess we should use ``is_true`` here.

Reply:
cool. Can you fix this?

Reply:
Yup, fixed in 7747f2aef3d1b32caeb415baadc3102d53831c52

Reply:
+1